scripts/obtain/dotnet-install.ps1

```diff
@@ -118,6 +118,11 @@ if ($SharedRuntime -and (-not $Runtime)) {
 $VersionRegEx="/\d+\.\d+[^/]+/"
 $OverrideNonVersionedFiles = !$SkipNonVersionedFiles
 
+# Default Tls protocol for .NET Framework 4.5 does not allow for Tls1.2.
+[System.Net.ServicePointManager]::SecurityProtocol = [System.Net.SecurityProtocolType]::Tls12;
```

Comment:
You shouldn't be changing all enabled protocols of the user here but rather `ADD` only the ones you need, like you do in line 124

Reply:
`+=` isn't safe to use if `Tls12` flag is already set. I guess the correct way to fix this is to use "binary or" `-bor`.